src/app/(admin)/new_dashboard/guides/[id]/page.tsx

```diff
@@ -1,54 +1,51 @@
 "use client";
+
+import React from "react";
 import PageContainer from "@/components/PageContainer";
 import { useParams } from "next/navigation";
 import { useGuide } from "../hooks";
 import DashboardCard from "../../shared/Card";
 import { Typography, CircularProgress, Alert, Box, Divider } from "@mui/material";
-import { Guide } from "@/api/types/Guide";
 import ReviewList from "../components/ReviewList";
-import AddReviewBox from "../components/AddReviewBox";
+import parse from "html-react-parser";
+import "quill/dist/quill.snow.css";
 
-const GuidePage = () => {
+const GuidePage: React.FC = () => {
   const params = useParams();
-
-  const id = params?.id ? Number(params.id) : null;
-  
   
-  let guide: Guide | undefined = undefined;
+  const id = params?.id ? Number(params.id) : null;
 
- const {data:response , isLoading, isError, error, isSuccess} = useGuide(id ?? 0);
+  const { data: response, isLoading, isError, error, isSuccess } = useGuide(id ?? 0);
 
- 
   if (isLoading) {
     return <CircularProgress />;
   }
 
-  if(isError) {
+  if (isError) {
     return <Alert severity="error">{error.message}</Alert>;
   }
 
-  if(isSuccess && "data" in response) {
-    guide = response.data;
-    if(!guide.title) {
+  if (isSuccess && "data" in response) {
+    const guide = response.data;
+    const creatorAndDateString = `Created by ${guide.creator?.username} on ${new Date(guide.createdAt).toLocaleDateString()}`;
```

Comment:
Why call the variable "String" if its already obvious. Instead you can call this "creationInfo".